packages/checkout/widgets-lib/src/widgets/bridge/BridgeWidget.tsx

```diff
@@ -80,7 +80,9 @@ export function BridgeWidget({
     {
       ...initialViewState,
       view: { type: BridgeWidgetViews.WALLET_NETWORK_SELECTION },
-      history: [{ type: BridgeWidgetViews.WALLET_NETWORK_SELECTION }],
+      history: initialViewState.history[initialViewState.history.length - 1].type === SharedViews.TOP_UP_VIEW
```

Comment:
Is initialViewState.history always going to have at least 1 item in there @deepti-imx ?